Bottom: 47142b4a6d42e76c81caf36b1b48b4a38063eaf9
Top:    77e6f9a64fe21d866c09b5e3cf02a3b39e104bc2
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 19:40:46 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 488ad1e..f1ec1e5 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -38,6 +38,7 @@
 - fix systemd unidir in spec file.
 - document browse option in man page.
 - fix some automount(8) typos.
+- syncronize handle_mounts() shutdown.
 
 25/07/2012 autofs-5.0.7
 =======================
